api_tests/auth/test_endpoint.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+import requests
+import json
+import config
+
+class TestEndpoint(unittest.TestCase):
+
+    @classmethod
+    def setUpClass (self):
+        config_dict  = config.parse()
+        self.API_URL = config_dict['url'] + '/auth'
+
+    def run_tests (self):
+        print '\nTesting /auth Endpoint:'
```

Comment:
If you are using unittest to run this, you can just set the docstring to a description of the test and it will print to the console, eliminating the need for all these print statements.